sysutils/flashrom/Portfile

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                flashrom
+version             1.0
+categories          sysutils
+platforms           darwin
+license             GPL-2
+maintainers         ra1nb0w openmaintainer
+description         universal flash programming utility
+long_description    utility for identifying, reading, writing, verifying and erasing flash chips.\
+                    It is designed to flash BIOS/EFI/coreboot/firmware/optionROM images on \
+                    mainboards, network/graphics/storage controller cards, and various other \
+                    programmer devices.
+homepage            https://www.flashrom.org/Flashrom
+master_sites        https://download.flashrom.org/releases/
+use_bzip2           yes
+
+checksums           rmd160  b2e0c62d8a0ae592a6d84b69c5f5ea3283dcfefe \
+                    sha256  3702fa215ba5fb5af8e54c852d239899cfa1389194c1e51cb2a170c4dc9dee64 \
+                    size    321693
+
+# needed until next version
+patchfiles          patch-libusb_set_debug-deprecated.diff
+
+depends_build       port:pkgconfig
+
+depends_lib         port:libftdi0 \
+                    port:libusb-compat \
+                    port:libusb
+
+use_configure       no
```

Comment:
When using this option, you need to make sure you are [using the right compiler](https://trac.macports.org/wiki/UsingTheRightCompiler). This at least means either disabling the universal variant or making changes to ensure that it works.

Reply:
I didn't know this behaviour. thanks for spotting out